server/webhook_parser.go

```diff
@@ -90,7 +94,7 @@ func ParseWebhook(bb []byte) (wh Webhook, err error) {
 	return wh, nil
 }
 
-func parseWebhookChangeLog(jwh *JiraWebhook) (Webhook, error) {
```

Comment:
Reverting decision to return/log error when none of the `fields` match our supported fields. We would be potentially spamming the logs with benign errors.

Reply:
Wasn't it in response to a customer wanting to know why the webhook wasn't being processed? Maybe we should just make the report at the info or debug level -- so it's there if we need it?

Reply:
> Wasn't it in response to a customer wanting to know why the webhook wasn't being processed?

Yes but the reason it was not being processed was not because of the unsupported `fields`, it was because of the unsupported `event type`. It may be useful to leave this here, but I think it was a premature decision to add this before we knew what was going on with the user's situation.

> Maybe we should just make the report at the info or debug level -- so it's there if we need it?

Sounds fine to me. Would the frequency of logging here be reasonable?

Reply:
@cpoile We don't have a pointer to the plugin here so we can't call `p.API.LogDebug`. What do you suppose we do? Do you know if there is usually a solution for logging in this sort of scenario?

Reply:
@mickmister yah, just ignore it for now. The server logs won't really be needed (we'll know what we need to add).

Reply:
@cpoile This may be something we want to solve for the future though. It's good to decouple code that doesn't need to have access to everything the `Plugin` struct has, but logging should be available anywhere.